cookbook/bundles/best_practices.rst

```diff
@@ -305,41 +294,60 @@ The end user can provide values in any configuration file:
 
         # app/config/config.yml
         parameters:
-            acme_hello.email.from: fabien@example.com
+            acme_blog.author.email: fabien@example.com
 
     .. code-block:: xml
 
         <!-- app/config/config.xml -->
         <parameters>
-            <parameter key="acme_hello.email.from">fabien@example.com</parameter>
+            <parameter key="acme_blog.author.email">fabien@example.com</parameter>
         </parameters>
 
     .. code-block:: php
 
         // app/config/config.php
-        $container->setParameter('acme_hello.email.from', 'fabien@example.com');
+        $container->setParameter('acme_blog.author.email', 'fabien@example.com');
 
-    .. code-block:: ini
+Retrieve the configuration parameters in your code from the container::
 
-        ; app/config/config.ini
-        [parameters]
-        acme_hello.email.from = fabien@example.com
+    $container->getParameter('acme_blog.author.email');
 
-Retrieve the configuration parameters in your code from the container::
+Even if this mechanism is simple enough, you should consider using the more
+advanced :doc:`semantic bundle configuration </cookbook/bundles/configuration>`.
 
-    $container->getParameter('acme_hello.email.from');
+Services
+--------
 
-Even if this mechanism is simple enough, you are highly encouraged to use the
-semantic configuration described in the cookbook.
+If the bundle defines services, they must be prefixed with the bundle alias.
+For example, AcmeBlogBundle services must be prefixed with ``acme_blog``.
 
-.. note::
+In addition, services not meant to be used by the application directly, should
+be :ref:`defined as private <container-private-services>`.
+
+.. seealso::
+
+    You can learn much more about service loading in bundles reading this article:
+    :doc:`How to Load Service Configuration inside a Bundle </cookbook/bundles/extension>`.
+
+Composer Metadata
+-----------------
+
+The ``composer.json`` file should include at least the following metadata:
```

Comment:
I would also add the ``autoload`` section. Symfony relies on the composer autoloader so shared bundles should register themselves in the autoloader